### DISCUSSION TITLE
DefaultProgressBar blinking in gameplay

### DISCUSSION THREAD

**Discussion by Firmatorenio** [ppy/osu#23757] [Q&A] [answered]
The bar in question seems to be randomly reinitializing during gameplay, as seen on the [video](https://www.youtube.com/watch?v=KFsc1ulWV2Y).

Some things I've tested around:
- happens across all gamemodes and on any map
- doesn't seem to be tied to specific spots in the maps (so, seems completely random and doesn't happen on the same spots across different plays)
- happens with auto too
- isn't specific to my custom skin and happens with the default triangles (2017) skin too
- happens on both ogl and dx11

For reference, I don't like the new argon progress bar and would rather not switch to it because it makes it difficult to judge the density of the parts of the map that are already completed.

[runtime.log](https://github.com/ppy/osu/files/11656051/runtime.log)

**Comment by bdach** [comment, marked as answer]
https://github.com/ppy/osu/issues/23760
